rskj-core/src/main/java/org/ethereum/util/ByteUtil.java

```diff
@@ -446,7 +392,7 @@ public static int getBit(byte[] data, int pos) {
         int posByte = data.length - 1 - pos / 8;
         int posBit = pos % 8;
         byte dataByte = data[posByte];
-        return Math.min(1, (dataByte & (1 << (posBit))));
```

Comment:
I'm worried there might be bugs in this method (`getBit`), and it's only used for testing. Could you move it there?

Reply:
I added this by Sonar recommendation, to force the correct type casting